Sources/Paywalls/PaywallData.swift

```diff
@@ -618,7 +623,50 @@ extension PaywallData {
 // MARK: - Constructors
 
 extension PaywallData {
+    #if PAYWALL_COMPONENTS
+    init(
+        templateName: String,
+        config: Configuration,
+        localization: [String: LocalizedConfiguration],
+        localizationByTier: [String: [String: LocalizedConfiguration]],
+        assetBaseURL: URL,
+        revision: Int = 0,
+        zeroDecimalPlaceCountries: [String] = [],
+        componentData: PaywallComponent.Data? = nil
+    ) {
+        self.templateName = templateName
+        self.config = config
+        self.localization = localization
+        self.localizationByTier = localizationByTier
+        self.assetBaseURL = assetBaseURL
+        self.revision = revision
+        self._zeroDecimalPlaceCountries = .init(apple: zeroDecimalPlaceCountries)
+        self.componentData = componentData
+    }
 
+    /// Creates a test ``PaywallData`` with one localization.
+    public init(
+        templateName: String,
+        config: Configuration,
+        localization: LocalizedConfiguration,
+        assetBaseURL: URL,
+        revision: Int = 0,
+        locale: Locale = .current,
+        zeroDecimalPlaceCountries: [String] = [],
+        componentData: PaywallComponent.Data? = nil
+    ) {
+        self.init(
+            templateName: templateName,
+            config: config,
+            localization: [locale.identifier: localization],
+            localizationByTier: [:],
+            assetBaseURL: assetBaseURL,
+            revision: revision,
+            zeroDecimalPlaceCountries: zeroDecimalPlaceCountries,
+            componentData: componentData
+        )
+    }
+    #else
```

Comment:
@jamesrb1 I think this can be removed now too since we aren't using `PaywallData` anymore for this?

Reply:
Gone!